plugins/ui/src/deephaven/ui/hooks/use_state.py

```diff
@@ -9,11 +9,11 @@
 
 
 @overload
-def use_state(initial_value: T) -> (T, Callable[[T], None]):
+def use_state(initial_value: T) -> tuple[T, Callable[[T], None]]:
     ...
 
 
-def use_state(initial_value: T | None = None) -> (T | None, Callable[[T], None]):
+def use_state(initial_value: T | None = None) -> tuple[T | None, Callable[[T], None]]:
```

Comment:
```suggestion
def use_state(initial_value: T | Callable[[], T] | None = None) -> tuple[T | None, Callable[[T], None]]:
```

Can you union the permitted Callable type for the param here while you're changing this line?

Reply:
I've also added the callable type to the setter function as well.